NMS-15147: Update docs to include RHEL 9 Install instructions (#5639)
OpenNMS/opennms: docs/modules/deployment/pages/core/centos-rhel9/postgresql.adoc

.Install language packs and set language
[source,console]
----
sudo dnf install langpacks-en glibc-all-langpacks -y
sudo localectl set-locale LANG=en_US.UTF-8
sudo localectl
sudo dnf makecache
sudo dnf update -y
sudo reboot
sudo dnf info postgresql-server
----

.Install PostgreSQL client and server
[source, console]
----
sudo dnf -y install https://download.postgresql.org/pub/repos/yum/reporpms/EL-9-x86_64/pgdg-redhat-repo-latest.noarch.rpm
sudo dnf -y install postgresql14-server
----

> .Initialize the PostgreSQL database
> [source, console]
> ----
> sudo /usr/pgsql-14/bin/postgresql-14-setup initdb
> ----

Reviewer: ```suggestion
.Initialize the PostgreSQL database
[source, console]
sudo /usr/pgsql-14/bin/postgresql-14-setup initdb
```

> .Enable PostgreSQL on system boot and start immediately
> [source, console]
> ----
> sudo systemctl enable --now postgresql-14
> ----

Reviewer: ```suggestion
.Enable PostgreSQL on system boot and start immediately
[source, console]
sudo systemctl enable --now postgresql-14
```

> .Create an opennms database user and password
> [source, console]
> ----
> sudo -i -u postgres createuser -P opennms
> ----

Reviewer: ```suggestion
.Create an opennms database user and password
[source, console]
sudo -i -u postgres createuser -P opennms
```

> IMPORTANT: You must provide a password for the `opennms` database user.
>          This guide uses `YOUR-OPENNMS-PASSWORD` as a placeholder.
>          Please set a secure password.

Reviewer: ```suggestion
IMPORTANT: You must provide a secure password for the `opennms` database user account.
This guide uses `YOUR-OPENNMS-PASSWORD` as a placeholder.
```

> .Create an empty database and set the owner to the opennms user
> [source, console]
> ----
> sudo -i -u postgres createdb -O opennms opennms
> ----

Reviewer: ```suggestion
.Create an empty database and set the owner to the opennms user
[source, console]
sudo -i -u postgres createdb -O opennms opennms
```

> .Set a password for PostgreSQL superuser
> [source, console]
> ----
> sudo -i -u postgres psql -c "ALTER USER postgres WITH PASSWORD 'YOUR-POSTGRES-PASSWORD';"
> ----

Reviewer: ```suggestion
.Set a password for PostgreSQL superuser
[source, console]
sudo -i -u postgres psql -c "ALTER USER postgres WITH PASSWORD 'YOUR-POSTGRES-PASSWORD';"
```

> IMPORTANT: Change `YOUR-POSTGRES-PASSWORD` to a secure one.
>          The superuser is required to be able to initialize and change the database schema for installation and updates.

Reviewer: ```suggestion
IMPORTANT: Change `YOUR-POSTGRES-PASSWORD` to a secure one.
The superuser is required to be able to initialize and change the database schema for installation and updates.
```

> 
> [[core-centos-rhel9-pg_hba]]
> .Change the access policy for PostgreSQL
> [source, console]
> ----
> sudo vi /var/lib/pgsql/data/pg_hba.conf
> ----

Reviewer: ```suggestion
[[core-centos-rhel9-pg_hba]]
.Change the access policy for PostgreSQL
[source, console]
sudo vi /var/lib/pgsql/data/pg_hba.conf
```

.Allow {page-component-title} to access the database over the local network with an MD5 hashed password
[source, pg_hba.conf]
----

> host    all             all             127.0.0.1/32            md5<1>
> host    all             all             ::1/128                 md5<1>

Reviewer: ```suggestion
host    all             all             127.0.0.1/32            md5 <1>
host    all             all             ::1/128                 md5 <1>
```

----

<1> Change method from `ident` to `md5` for IPv4 and IPv6 on localhost.

> .Apply configuration changes for PostgreSQL
> [source, console]
> ----
> sudo systemctl reload postgresql
> ----

Reviewer: ```suggestion
.Apply configuration changes for PostgreSQL
[source, console]
sudo systemctl reload postgresql
```
